File: gris.py
import matplotlib.pyplot as plt
import numpy as np

def luminance(Img):
    return np.mean(Img)

def constrast(Img):
    return np.var(Img)

def profondeur(Img):
    return np.max(Img)

def Ouvrir(Img):
    return Img

File: matlab/gris.m
function lum = luminance(Img)
    lum = mean(Img(:));
end

function c = constrast(Img)
    c = var(double(Img(:)));
end

function p = profondeur(Img)
    p = max(Img(:));
end

function ImgOut = Ouvrir(Img)
    ImgOut = Img;
end
